### ISSUE TITLE
Copy File to NAS

### ISSUE THREAD

**Issue by Marcello66** [MattVador/node-red-contrib-syno#10]
Hi,
I want to copy a CSV file from RasperryPi with Node-Red to my Synology NAS.
How do I do it? Does anyone have an example?

Thanks in advance.

**Comment by GeorgeCastanza** [comment]
Did you figure this out?  I need a working example.

**Comment by Marcello66** [comment]
No, I didn't find out. But I didn't have time for that either. I also need a working example.

**Comment by MattVador** [comment]
Hi Marcello66, it seems that the "upload" function of the syno package does not work. I tried to analyze it but it seems impossible as it is developed.

Maybe you can mount a NAS directory on the Raspberry and use the read and write file nodes to make the copy.

**Comment by UweGi** [comment]
Is there now an example to upload/download a file with DS File to Synology Station?

**Comment by My-Random-Thoughts** [comment]
There is now - https://flows.nodered.org/flow/5d24159af4001a9a34110d8f71f954af
